.github/workflows/ci-alternative-runtime.yml

```diff
@@ -35,6 +35,9 @@ jobs:
           - runtime: nsolid
             node-version: 20
             nsolid-version: 5
+          - runtime: nsolid
+            node-version: 22
```

Comment:
Not sure:

> The N|Solid Runtime is our enhanced Open Source build of the current LTS versions of Node. It keeps in step with upstream Node.js releases, currently tracking the Node.js v18.x Hydrogen LTS and Node.js v20.x Iron LTS release lines.

https://docs.nodesource.com/docs/nsolid/runtime/overview/

I think we should just wait their input TBH